### DISCUSSION TITLE
Any Documentation for Funkin Modules yet

### DISCUSSION THREAD

**Discussion by lazyremixmanordummyplayinggmod** [FunkinCrew/Funkin#2231] [Q&A]
Just asking bc lol it never released with proper documentation any fan made ones or crap ik it has been a day but I have seen people do stuff and also make scripts and shit

**Comment by biomseed** [comment]
I'm pretty sure that's coming in the near future
Right now you need to guess pretty much

**Reply by ChandelureF-5** [reply]
which is honestly a BIT annoying... but generally not too bad, ig

**Comment by Lasercar** [comment]
https://funkincrew.github.io/funkin-modding-docs/
